src/components/chart/_macro-options.md

```diff
@@ -36,19 +36,21 @@
+| labelFormat  | string | false    | A format string for the axis label. Examples of string formats can be found in these [docs](https://www.highcharts.com/docs/chart-concepts/templating). |
+| tickInterval | number | false    | The interval of the tick marks in axis units. Useful when you want to space out the labels (e.g. show every 2nd or 5th label).                          |
 
 ### X_Axis
 
-| Name        | Type   | Required | Description                                                                                                                                               |
-| ----------- | ------ | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| title       | string | false    | The title text displayed on the x-axis                                                                                                                    |
-| labelFormat | string | false    | A format string for the x-axis label. Examples of string formats can be found in these [docs](https://www.highcharts.com/docs/chart-concepts/templating). |
-| categories  | array  | false    | Labels for each tick mark along the x-axis.                                                                                                               |
-| type        | string | false    | The type of axis. Can be one of `linear`, `logarithmic`, `datetime` or `category`. Defaults to linear.                                                    |
+| Name         | Type   | Required | Description                                                                                                                                               |
+| ------------ | ------ | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| title        | string | false    | The title text displayed on the x-axis                                                                                                                    |
+| labelFormat  | string | false    | A format string for the x-axis label. Examples of string formats can be found in these [docs](https://www.highcharts.com/docs/chart-concepts/templating). |
+| categories   | array  | false    | Labels for each tick mark along the x-axis.                                                                                                               |
+| type         | string | false    | The type of axis. Can be one of `linear`, `logarithmic`, `datetime` or `category`. Defaults to linear.                                                    |
+| tickInterval | number | false    | The interval of the tick marks in axis units. Useful when you want to space out the labels (e.g. show every 2nd or 5th label).                            |
```

Comment:
We need this to be `tickIntervalMobile` and `tickIntervalDesktop` so that the editor can configure it differently for both. You can then update the options in https://github.com/ONSdigital/design-system/blob/main/src/components/chart/common-chart-options.js#L144-L176 to add the mobile tickInterval.

Reply:
updated